kafka/build_static.go

```diff
@@ -3,5 +3,6 @@
 
 package kafka
 
-// #cgo pkg-config: rdkafka-static
+// #cgo pkg-config: rdkafka
+// #cgo LDFLAGS: -Wl,-Bstatic -lrdkafka -llz4 -lssl -lcrypto -lz -lgcc -Wl,-Bdynamic -lsasl2 -lrt -ldl -lm -lpthread
```

Comment:
This list of libraries depends on the librdkafka build options, which will vary and is thus not general applicable.
That's why we use pkg-config, it tells us the libraries librdkafka was built with